packages/components/src/components/block/block.tsx

```diff
@@ -97,7 +97,7 @@ export class Block extends LitElement implements InteractiveComponent {
   /**
    * When `true`, the component displays a draggable button.
    *
-   * @deprecated No longer necessary. Use Block Group for draggable functionality.
+   * @deprecated in v3.0, removal target v6.0 - No longer necessary. Use Block Group for draggable functionality.
```

Comment:
Consistency and copyedit suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - No longer necessary. Use `calcite-block-group` for draggable functionality.
```